pages/task/read/views/components/activity-log.jsx

```diff
@@ -1,43 +1,45 @@
 import React, { Fragment, useState } from 'react';
 import classnames from 'classnames';
 import get from 'lodash/get';
-import { Link, Snippet, Markdown } from '@asl/components';
+import { Link, Snippet, Markdown, Inset } from '@asl/components';
 import { dateFormat } from '../../../../../constants';
 import format from 'date-fns/format';
+import PplDeclarations from './ppl-declarations';
 
-const getProfileLink = ({ id, name, establishmentId, asruUser }) => {
+function ProfileLink({ id, name, establishmentId, asruUser }) {
```

Comment:
I like this, if it looks like a component, make it a component 👍

Reply:
I agree. Nice.